### DISCUSSION TITLE
Any plan for 5.2 yet? (I know it's still preview, but it's very stable for our production.)

### DISCUSSION THREAD

**Discussion by onyxlee** [microdee/OpenUnrealSpaceMouse#88] [General]
Hi dear David, you are our savior! Your update is like my lifeline. Without it, my wrist starts hurting me very quickly.

Could you compile a 5.2 Preview for us? I know UE marketplace doesn't accept it just yet, but you can upload it here.

Thank you so much!!!

**Comment by MarkEHawkins** [comment]
Hi David, I'm sorry if its been asked before. Is there anything the Spacemouse users can do to together to update the 5.2 version.  Maybe if there is detailed instruction sheet on how to compile to update to 5.2. Is it because the the plugin code needs to change on every release. 
Its hard not to be swept along the UE5 releases, there are some great new updates. Thanks for your hard work, I would be happy to purchase the app again when the 5.2 is listed on the Unreal App store.

**Comment by microdee** [comment]
can you try latest release now for 5.2? https://github.com/microdee/UE4-SpaceMouse/releases/latest
